smrf/tests/check_mixin.py

```diff
@@ -36,3 +38,61 @@ def test_snow_density(self):
 
     def test_vapor_pressure(self):
         self.compare_netcdf_files('vapor_pressure.nc')
+
+
+class CheckSMRFOutputRatios(object):
+    """Check the SMRF test case for all the variables. To be used as a
+    mixin for tests to avoid these tests running more than once.
+
+    Example:
+        TestSomethingNew(CheckSMRFOutputRatios, SMRFTestCase)
+            RATIO_MAP = {
+                'air_temp': 1.0,
+                'precip': 2.0,
+            }
+    """
+    RATIO_MAP = {}
+    DECIMAL = 4
+
+    def _assert_ratio(self, file_name, variable):
```

Comment:
Methods with an underscore in Python are considered private to the class. I agree with the use here since the ‘public’ ones are testing the variables and have a helper method internal to the class.